tests/federation_rooms_invite_test.go

```diff
@@ -149,6 +149,23 @@ func TestFederationRoomsInvite(t *testing.T) {
 			alice.MustSyncUntil(t, client.SyncReq{Filter: includeLeaveSyncFilter}, client.SyncLeftFrom(bob2.UserID, roomID))
 		})
 
+		t.Run("Inviter user can rescind invite over federation", func(t *testing.T) {
+			t.Parallel()
+			roomID := alice.MustCreateRoom(t, map[string]interface{}{
+				"preset": "private_chat",
+				"invite": []string{bob.UserID},
+			})
+			bob.MustSyncUntil(t, client.SyncReq{}, client.SyncInvitedTo(bob.UserID, roomID))
+			alice.MustDo(t, "POST", []string{"_matrix", "client", "v3", "rooms", roomID, "kick"},
```

Comment:
It would be good to have a test where someone else rescinds the invite (someone different than the inviter)